annotator-core/src/main/java/edu/ucr/cs/riple/core/Annotator.java

```diff
@@ -229,6 +229,8 @@ private void forceResolveRemainingErrors() {
     Utility.buildTarget(config);
     Set<Error> remainingErrors =
         Utility.readErrorsFromOutputDirectory(config, config.target, fieldDeclarationStore);
+    System.out.println(
+        remainingErrors.size() + " errors remaining. Applying suppression annotations.");
```

Comment:
Is this print supposed to be part of this PR?

Reply:
@msridhar Yes, I was planning to add all changes to error suppression in this PR, but it is already divided into two PRs. I can add this in a follow up PR.